PiWriteBack.Test/ConsoleWrapper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace Cognite.PiWriteBack.Test
+{
+    public class ConsoleWrapper : IDisposable
```

Comment:
This class is now included in the `Cognite.Extractor.Testing` library, if you prefer.